definitions/divorce/json/CaseEventToFields/CaseEventToFields.json

```diff
@@ -4190,7 +4190,7 @@
     "PageID": "refusalRejection",
     "PageLabel": "Refusal order: Reject",
     "PageDisplayOrder": 3,
-    "FieldShowCondition": "RefusalRejectionReasonCONTAINS\"other\"",
+    "FieldShowCondition": "DecreeNisiGranted=\"No\" AND RefusalRejectionReasonCONTAINS\"other\"",
```

Comment:
is this correct? it doesn't need space before `CONTAINS`.

Reply:
does it parse for `CONTAINS` and just takes whatever comes before? this is so... not good

Reply:
Yes it looks strange but thats the format that we need to follow in order to do  contains check